docs/notifications.json

```
@@ -8,5 +8,15 @@
   {
     "id": 53,
     "text": "You can <a style=\"color: inherit;\" target=\"_blank\" rel=\"noopener\" href=\"https://twitter.com/MUI_hq\">follow us on Twitter</a> to receive exclusive tips and updates about MUI and the React ecosystem."
+  },
+  {
+    "id": 56,
+    "title": "🚀 Row Grouping feature - Data Grid",
+    "text": "Released in <code>@mui/x-data-grid v5.3.0</code>. <a style=\"color: inherit;\" data-ga-event-category=\"Blog\" data-ga-event-action=\"notification\" data-ga-event-label=\"introducing-the-row-grouping-feature\" href=\"/blog/introducing-the-row-grouping-feature/\">Read the blog post</a> or <a style=\"color: inherit;\" target=\"_blank\" rel=\"noopener\" href=\"https://github.com/mui-org/material-ui-x/releases/tag/v5.3.0\">the changelog</a> for more details."
```

```suggestion
    "text": "Released in <code>@mui/x-data-grid-pro v5.3.0</code>. <a style=\"color: inherit;\" data-ga-event-category=\"Blog\" data-ga-event-action=\"notification\" data-ga-event-label=\"introducing-the-row-grouping-feature\" href=\"/blog/introducing-the-row-grouping-feature/\">Read the blog post</a> or <a style=\"color: inherit;\" target=\"_blank\" rel=\"noopener\" href=\"https://github.com/mui-org/material-ui-x/releases/tag/v5.3.0\">the changelog</a> for more details."
```

It's only available in Pro package (temporarily, until we introduce Premium plan).